config/_default/menus/menus.en.yaml

```
@@ -2703,124 +2703,130 @@ main:
     identifier: synthetics_dashboards_testing_coverage
-    weight: 803
+    weight: 603
   - name: Test Summary
     url: synthetics/dashboards/test_summary
     parent: synthetics_dashboards
     identifier: synthetics_dashboards_test_summary
-    weight: 804
+    weight: 604
   - name: APM Integration
     url: synthetics/apm/
     parent: synthetics
     identifier: synthetics_apm
-    weight: 9
+    weight: 7
   - name: Settings
     url: synthetics/settings/
     parent: synthetics
-    weight: 10
+    weight: 8
   - name: Metrics
     url: synthetics/metrics/
     parent: synthetics
-    weight: 11
+    weight: 9
   - name: Data Security
     url: data_security/synthetics/
     parent: synthetics
-    weight: 12
+    weight: 10
   - name: Guides
     url: synthetics/guide/
     parent: synthetics
     identifier: synthetics_guides
-    weight: 13
+    weight: 11
   - name: Troubleshooting
     url: synthetics/troubleshooting/
     parent: synthetics
     identifier: synthetics_troubleshooting
-    weight: 14
+    weight: 12
+  - name: Continuous Testing
+    url: continuous_testing/
+    pre: synthetics
+    identifier: continuous_testing
+    parent: ux_monitoring_heading
+    weight: 200000
+  - name: CI/CD Integrations
+    url: continuous_testing/cicd_integrations
+    parent: continuous_testing
+    identifier: continuous_testing_cicd_integrations
+    weight: 1
+  - name: Configuration
+    url: /continuous_testing/cicd_integrations/configuration
+    parent: continuous_testing_cicd_integrations
+    identifier: continuous_testing_cicd_integrations_configuration
+    weight: 101
+  - name: Azure DevOps Extension
+    url: /continuous_testing/cicd_integrations/azure_devops_extension
+    parent: continuous_testing_cicd_integrations
+    identifier: continuous_testing_cicd_integrations_azure_devops_extension
+    weight: 102
+  - name: CircleCI Orb
+    url: /continuous_testing/cicd_integrations/circleci_orb
+    parent: continuous_testing_cicd_integrations
+    identifier: continuous_testing_cicd_integrations_circleci_orb
+    weight: 103
+  - name: GitHub Actions
+    url: /continuous_testing/cicd_integrations/github_actions
+    parent: continuous_testing_cicd_integrations
+    identifier: continuous_testing_cicd_integrations_github_actions
+    weight: 104
+  - name: GitLab
+    url: /continuous_testing/cicd_integrations/gitlab
+    parent: continuous_testing_cicd_integrations
+    identifier: continuous_testing_cicd_integrations_gitlab
+    weight: 105
+  - name: Jenkins
+    url: /continuous_testing/cicd_integrations/jenkins
+    parent: continuous_testing_cicd_integrations
+    identifier: continuous_testing_cicd_integrations_jenkins
+    weight: 106
+  - name: Testing Tunnel
+    url: /continuous_testing/cicd_integrations/testing_tunnel
+    parent: continuous_testing_cicd_integrations
+    identifier: continuous_testing_cicd_integraions_testing_tunnel
+    weight: 107
+  - name: CI Results Explorer
```

```suggestion
  - name: Explorer
```
We're planning to rename the left nav to `Explorer` soon